CHANGELOG.md

```diff
@@ -1,4 +1,13 @@
-##2015-08-12 - Supported Release 1.1.1
+##2015-08-13 - SUpported Release 1.1.2
+###Summary
+
+Fix critical bug when writing dword and qword values 
+
+####Bugfixes
+- Fix the way we write dword and qword values
+- Change the way we convert to byte arrays
+
+##~~2015-08-12 - Supported Release 1.1.1~~ - Deleted
```

Comment:
the old release didn't mention the summary or details of what the fix for that was about.